README.md

```diff
@@ -20,16 +20,129 @@ def deps do
 end
 ```
 
-## Usage
+## Demos
 
 The `examples` directory shows how to send and receive streams from a web browser.
-There are following two demos there:
+There are following three demos there:
+* `live_view` - simple Phoenix project using `Membrane.WebRTC.Live.Player` and `Membrane.WebRTC.Live.Capture` to echo video stream
```

Comment:
Please add a note saying that it uses live_view as well.

Reply:
```suggestion
* `live_view` - simple Phoenix LiveView project using `Membrane.WebRTC.Live.Player` and `Membrane.WebRTC.Live.Capture` to echo video stream
```